src/multi_vector_simulator/C0_data_processing.py

```diff
@@ -1774,7 +1775,19 @@ def process_maximum_cap_constraint(dict_values, group, asset, subasset=None):
         asset_dict.update({MAXIMUM_CAP: {VALUE: None}})
     else:
         if asset_dict[MAXIMUM_CAP][VALUE] is not None:
+<<<<<<< HEAD
             # set maximumCap to None if it is smaller than installedCap
+=======
+            # adapt maximumCap of non-dispatchable sources
+            if group == ENERGY_PRODUCTION and asset_dict[FILENAME] is not None:
+                asset_dict[MAXIMUM_CAP][VALUE] = (
+                        asset_dict[MAXIMUM_CAP][VALUE] * asset_dict[TIMESERIES_PEAK][VALUE]
+                )
+                logging.debug(
+                    f"Parameter {MAXIMUM_CAP} of asset '{asset_dict[LABEL]}' was multiplied by the peak value of {TIMESERIES}. This was done as the aimed constraint is to limit the power, not the flow."
+                )
+            # check if maximumCap is greater that installedCap
+>>>>>>> Create function (C1.energy_system_max_demand_fulfillment)
```

Comment:
The changes in this commit are not quite clear - I think you mixed `black` indentation changes and the changes of the commit itself, and there is a changelog entry in here. We can ignore those, I guess. But there is also a merge conflict comitted! Make sure that this is removed in later commits.

Reply:
I saw that you fixed it by removing all lines (see below). Not sure if that is right, though?